### PR TITLE
feat: add to cart and preview

### PR DESCRIPTION
**Pull request by Komitaji**

### CODE REVIEW

File: src/pages/review-order.tsx [hunk 1]
```diff
@@ -1,16 +1,239 @@
...
+    client: client,
+  });
+  useEffect(() => {
+    if (loading == false && !error && data) {
+      const newCampaigns: any[] = [];
+      Promise.all(
+        data.buyerCampaigns.map(async (buyerCampaign: CampaignData) => {
+          const url = formatIpfsUri(buyerCampaign.uri);
+          const uriData = await (await fetch(url)).json();
+          // if (convertOldFormats(format) !== convertOldFormats(uriData.format))
```
Comment (Komitaji) [inline src/pages/review-order.tsx:223]: @Srile I couldn't from where the `format` is supposed to be taken from. Hence I have commented this part out which you can uncomment.

**Comment by JSeam2** [comment]
Some issues with the checkout flow

1. You select any format for the ad-space, for example a mix of tall, wide, square, but then in the checkout you select a square campaign. A consequence of this is that the ad asset that is shown in the game will not be shown properly.

2. I should be able to switch between states of the checkout tab on the right of the page by clicking on the whole object when it is collapsed. Clicking on `<` can be very tricky for users. When it is expanded clicking on `>` will then collapse the checkout. To make it easier to click on it might be better to use a button component. Though this might mess up with styling.

**Comment by Srile** [comment]
If the user has not connected their wallet when pressing 'buy now', the wallet connection prompt should open up.

**Comment by Srile** [comment]
Small design issues on the checkout page:
image

1) Button font is not matching the designs
2) horizontal spacing for the collapsable should be aligned with other elements (button & header). Additionally, the collapsable should be more visually elevated from the background (maybe a darker shade of the bg color)

**Pull request merged**